R/pkg/R/mllib.R

```diff
@@ -53,26 +53,27 @@ setClass("AFTSurvivalRegressionModel", representation(jobj = "jobj"))
 #' @note KMeansModel since 2.0.0
 setClass("KMeansModel", representation(jobj = "jobj"))
 
-#' Saves the machine learning model to the input path
+#' Saves the MLlib model to the input path
```

Comment:
The ```write.ml``` can only be used for saving MLlib models, it can not save other machine learning model produced by native R functions. So I think the current description is accurate enough.